packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -77,29 +77,16 @@ $-chevron-expand-more: map-merge(
   line-height: line-height($theme-footer-font-family, 2);
 }
 
-.usa-footer__primary-link a,
-.usa-footer__secondary-link a {
-  text-decoration: none;
-
-  &:hover {
-    text-decoration: underline;
-  }
-}
-
 .usa-footer__primary-link {
   @include u-padding-x($theme-site-margins-mobile-width);
   @include u-padding-y(2);
-  @include u-text("ink", "no-underline", "bold");
+  @include u-text("ink", "bold");
```

Comment:
Removing `no-underline` from `u-text()` mixin properly applies color and weight without causing any unintentional regressions.